docs/agents/index.md

```diff
@@ -1,3 +1,7 @@
+---
+version: 1.0.0
```

Comment:
Is this going to be visible on the page? Also: because this number looks like a version number for ADK, I think at least some people are going to assume that it corresponds with a ADK version number, in which case this makes it look like the docs are way out of date. Maybe it's better to display a last updated date and keep the version as hidden metadata?